CHANGELOG.md

```diff
@@ -5,7 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-<!-- ## Unreleased -->
+## Unreleased
+- When injecting html imports into documents for inlining, bundler now tries
```

Comment:
I guess the notion of `injecting` is outside what users might conceive of.  I'll reword.

A note: evaluation ordering of imports can not be 100% preserved in cases where merge strategies build shared bundles -- I could contrive examples where it wont be, in tests, but not in this PR (like document X having 5 links to files that have since been moved into a shared bundle Y, now document X just links to shared bundle Y once and the order of links in Y is unknown).  The aim is just to preserve the order where it has been expressed in the dependency graph.